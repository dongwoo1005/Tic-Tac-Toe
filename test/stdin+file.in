game stdin movesO2.txt
C
NE
W
NW
N
quit
