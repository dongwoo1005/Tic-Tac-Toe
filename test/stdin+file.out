X's move
O's move
(plays S)
X's move
O's move
(plays SW)
X's move
O's move
(plays SE)
O wins
Score is
X 0
O 1
